### PR TITLE
Decompile sub_8002DB4

### PR DESCRIPTION
**Pull request by easyaspi314**
Maybe I did something useful this time.

### CODE REVIEW

**Comment by easyaspi314** [comment]
Done with this for now, might work on the rest of asm/rom8002D6C.s in a later PR.

It doesn't appear to be that difficult, there is just a lot of it.

File: include/declarations.h [hunk 1]
```diff
@@ -15,6 +15,12 @@ extern void (*gUnknown_0811DFFC[6])(struct AnimationStruct *, struct CourtScroll
 extern u32 * gFlagPtrs[3]; 
 extern s16 gSineTable[256+64];
 extern u32 gUnknown_080150D0[12];
+extern const u16 gUnknown_08015100[0x78/2];
+extern const u16 gUnknown_08015178[0x64/2];
+extern const u16 gUnknown_080151DC[0xDC/2];
+extern const u16 gUnknown_080152B8[0x28/2];
+extern const u16 gUnknown_080152E0[0x64/2];
+extern const u16 gUnknown_08015344[0x78/2];
```
Comment (atasro2) [inline include/declarations.h:23]: These shouldn't be u16  as they are examination data and there already exists a struct for these in the code called `ExaminationData`

File: src/rom8002D6C.c [hunk 2]
```diff
@@ -11,4 +14,191 @@ void sub_8002D70(struct Main * main)
     DmaCopy16(3, gUnknown_080150D0, gMain.roomData, sizeof(gUnknown_080150D0));
     DmaCopy16(3, gUnknown_08014FB8, gTalkData, sizeof(gUnknown_08014FB8));
     main->currentRoomId = 0;
-}
\ No newline at end of file
+}
+
+#define COPY_EXAM_DATA(data) DmaCopy16(3, data, gExaminationData, sizeof(data))
```
Comment (atasro2) [inline src/rom8002D6C.c:19]: i don't know about the name of this macro it loads that rooms examination data so COPY_EXAM_DATA sounds kinda misleading if you don't know what you're looking at i assume

**Review by atasro2 — CHANGES_REQUESTED** [review]

**Comment by atasro2** [comment]
Obsoleted by #62

**Pull request closed without merging**